backend/app.js

```diff
@@ -25,11 +27,27 @@ app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 app.use(cors())
 
+// use sessions for tracking logins
+app.use(session({
+  secret: 'home2health secret key',
```

Comment:
this is okay for now but this is a really sensitive part of your app that should be in .env

If other people get hold of this they can create any cookie they want and be authenticated as anyone